MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,1809 @@ This page lists the keys for each control and the element to which it is mapped
     </tr>
 </table>
 
+## SfImageEditor
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+
+    <tr>
+        <td rowspan="19">
+            SfImageEditorStyles  
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfImageEditorTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, it is possible to customize the appearance of the SfImageEditor without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.XForms.Themes;assembly=Syncfusion.Core.XForms"
+             ...>
+<Application.Resources>
+    <syncCore:SyncfusionThemeDictionary>
```

Comment:
resource node is not correct use code from application

https://github.com/syncfusion-content/maui-docs/pull/1958

Reply:
Updated